src/model/HeatExchangerAirToAirSensibleAndLatent.cpp

```diff
@@ -534,6 +539,59 @@ namespace model {
     setThresholdTemperature(1.7);
 
     setEconomizerLockout(true);
+
+    TableIndependentVariable var(model);
+    var.setInterpolationMethod("Linear");
+    var.setExtrapolationMethod("Linear");
+    var.setMinimumValue(0.0);
+    var.setMaximumValue(10.0);
+    var.setUnitType("Dimensionless");
+    var.addValue(0.75);
+    var.addValue(1.0);
+
+    TableLookup s75heating(model);
+    s75heating.setNormalizationMethod("DivisorOnly");
+    s75heating.setNormalizationDivisor(0.76);
+    s75heating.setMinimumOutput(0.0);
+    s75heating.setMaximumOutput(10.0);
+    s75heating.setOutputUnitType("Dimensionless");
+    s75heating.addOutputValue(0.81);
+    s75heating.addOutputValue(0.76);
+    s75heating.addIndependentVariable(var);
+    setSensibleEffectivenessofHeatingAirFlowCurve(s75heating);
+
+    TableLookup l75heating(model);
+    l75heating.setNormalizationMethod("DivisorOnly");
+    l75heating.setNormalizationDivisor(0.68);
+    l75heating.setMinimumOutput(0.0);
+    l75heating.setMaximumOutput(10.0);
+    l75heating.setOutputUnitType("Dimensionless");
+    l75heating.addOutputValue(0.73);
+    l75heating.addOutputValue(0.68);
+    l75heating.addIndependentVariable(var);
+    setLatentEffectivenessofHeatingAirFlowCurve(l75heating);
+
+    TableLookup s75cooling(model);
+    s75cooling.setNormalizationMethod("DivisorOnly");
+    s75cooling.setNormalizationDivisor(0.76);
+    s75cooling.setMinimumOutput(0.0);
+    s75cooling.setMaximumOutput(10.0);
+    s75cooling.setOutputUnitType("Dimensionless");
+    s75cooling.addOutputValue(0.81);
+    s75cooling.addOutputValue(0.76);
+    s75cooling.addIndependentVariable(var);
+    setSensibleEffectivenessofCoolingAirFlowCurve(s75cooling);
+
+    TableLookup l75cooling(model);
+    l75cooling.setNormalizationMethod("DivisorOnly");
+    l75cooling.setNormalizationDivisor(0.68);
+    l75cooling.setMinimumOutput(0.0);
+    l75cooling.setMaximumOutput(10.0);
+    l75cooling.setOutputUnitType("Dimensionless");
+    l75cooling.addOutputValue(0.73);
+    l75cooling.addOutputValue(0.68);
+    l75cooling.addIndependentVariable(var);
+    setLatentEffectivenessofCoolingAirFlowCurve(l75cooling);
```

Comment:
Ok, so I get why this is here: to maintain the same effectiveness at different airflows as we had before.

* We could also not do this, similar to what E+ would do if you create this object in the IDF Editor
    * Leave them empty so the effectiveness is linear.
    * You could extract that bit of code into a public method, something like "applyDefaultEffectivenessCurves" or something for convenience, so that the user has the ability to use the former defaults with a single method call.
    * VT (already done here) + modification of the OS resources ruby tests would ensure we don't show EUI deviations.
* If we want these curves to be set, then please name the Table / Vars